prov/shm/src/smr_rma.c

```diff
@@ -145,9 +121,13 @@ ssize_t smr_generic_rma(struct fid_ep *ep_fid, const struct iovec *iov,
 	if (ret)
 		return ret;
 
+	if (ep->util_ep.domain->mr_mode & FI_MR_VIRT_ADDR &&
+	    rma_count == 1 && !(op_flags & OFI_REMOTE_CQ_DATA))
+		cmds = 1;
```

Comment:
You can use:
```
cmds = 1 +( (mr_mode & VIRT_ADDR) && (rma_count == 1) && !(op_flags & CQ_DATA)
```
This eliminates the need to initialize cmds and the branch from the if statement.  Same comment applies further down